src/busterbunny.js

```diff
@@ -70,14 +70,25 @@ module.exports = (function() {
         };
 
         self.raiseEvents = function (eventId, event, options, afterRaised) {
-            _stats.queuedEventsToRaise++;
-            if (options instanceof Function) {
+            if(options instanceof Function && afterRaised)
+                throw new Error('the third argument must be an object when a fourth argument is provided.');
+
+            if (options instanceof Function && !afterRaised) {
                 afterRaised = options;
                 options = null;
             }
 
-            var eventData = self.encoder.encodeEvent(event);
-            publishOrQueue(config.exchange, eventId, eventData, options, afterRaised);
+            if(!(afterRaised instanceof Function))
```

Comment:
Shall I apply them to all the other places where there aren't braces?  I was applying the standard of no braces around single statements that cause the current execution to stop (return/throw).  I can apply this standard or put braces everywhere regardless.  What standard do you want to use?